cmd/gh/main.go

```diff
@@ -170,15 +171,26 @@ func mainRun() exitCode {
 	rootCmd.ValidArgsFunction = func(cmd *cobra.Command, args []string, toComplete string) ([]string, cobra.ShellCompDirective) {
 		var results []string
 		if aliases, err := cfg.Aliases(); err == nil {
-			for aliasName := range aliases.All() {
+			for aliasName, aliasValue := range aliases.All() {
 				if strings.HasPrefix(aliasName, toComplete) {
-					results = append(results, aliasName)
+					s := fmt.Sprintf("%s\tAlias for %q", aliasName, aliasValue)
+					results = append(results, s)
 				}
 			}
 		}
 		for _, ext := range cmdFactory.ExtensionManager.List() {
 			if strings.HasPrefix(ext.Name(), toComplete) {
-				results = append(results, ext.Name())
+				if ext.IsLocal() {
+					s := fmt.Sprintf("%s\tLocal extension gh-%s", ext.Name(), ext.Name())
+					results = append(results, s)
+				} else {
+					path := ext.URL()
+					if u, err := url.Parse(ext.URL()); err == nil {
+						path = u.Path[1:]
```

Comment:
Please let's also strip the `.git` suffix from the path, since most git remote URLs will have that.